etc/firebase-admin.api.md

```diff
@@ -390,6 +400,7 @@ export namespace auth {
         tokensValidAfterTime?: string;
         uid: string;
     }
+    {};

```

Comment:
Any idea what this is?

Reply:
None. (I just ran the tool.) I'll remove it and see if anything complains.

Reply:
Strange. Also removing it seems to cause a build break. Let's just add it for now. We can follow up with the API Extractor project separately to see what's going on (looks like a bug in the tool itself).